src/store/useConnectionStore.ts

```diff
@@ -97,7 +97,7 @@ const store: StateCreator<ConnectionStore> = (set, get) => ({
 					});
 
 					LoggerService.error(namespaces.stores.connectionCheckerStore, logeExtended);
-
+					resetChecker();
```

Comment:
Why `resetChecker()` in [Connections Table](https://github.com/autokitteh/web-platform/blob/c51ad4158e918bdb971ba98e049cd74a068a78d7/src/components/organisms/connections/table.tsx#L108) doesn't work?
We should check what the problem is and not try to find a workaround.